packages/core/src/server/devServer.ts

```diff
@@ -345,6 +345,7 @@ export async function createDevServer<
       await options.context.hooks.onAfterStartDevServer.call({
         port,
         routes,
+        environments: options.context.environments,
```

Comment:
Should we always put environments at the second parameter? To keep consistent

Reply:
Sounds reasonable, but some hooks don't have a first parameter, which is weird if for consistency...

Reply:
get, first param is ok